examples/hitl/rearrange_v2/world.py

```diff
@@ -18,6 +27,12 @@
 from habitat_sim.scene import SemanticRegion
 
 
+@dataclass
+class ObjectStateInfo:
+    state_spec: ObjectStateSpec
+    value: Any
```

Comment:
Out of curiosity, why is the value `Any` here?

Reply:
Object states can hold any type of value. We are currently only using booleans, but that may not be the case in the future.